node/src/test/kotlin/net/corda/node/utilities/X509UtilitiesTest.kt

```
@@ -208,7 +212,7 @@ class X509UtilitiesTest {
         serverCertAndKey.certificate.isValidOn(Date())
         serverCertAndKey.certificate.isSignatureValid(JcaContentVerifierProviderBuilder().build(caCertAndKey.certificate.subjectPublicKeyInfo))
 
-        assertTrue { serverCertAndKey.certificate.subject.toString().contains(MEGA_CORP.name.commonName) }
+        assertTrue { serverCertAndKey.certificate.subject.toString().contains(MEGA_CORP.name.organisation!!) }
```

Had a thought... as organisation is now a required field, perhaps the helper value should simply throw an exception (`IllegalStateException` because it means somehow an earlier test has failed) if none is found, and one of these then need to do `!!`?